index.js

```diff
@@ -71,6 +71,16 @@ const factory = options => {
     opts.issWhitelist.indexOf(token.payload.iss) > -1 ||
     reject(new IssWhitelistError(`iss '${token.payload.iss}' not in issWhitelist`))
 
+  const checkExp = ({ payload }) => {
+    const { exp } = payload
```

Comment:
What if `exp` is not present? Does this change effectively require the `exp` field?